libs/partners/anthropic/langchain_anthropic/chat_models.py

```diff
@@ -541,9 +541,6 @@ class Joke(BaseModel):
         populate_by_name=True,
     )
 
-    _client: anthropic.Client = PrivateAttr(default=None)  # type: ignore[assignment]
-    _async_client: anthropic.AsyncClient = PrivateAttr(default=None)  # type: ignore[assignment]
-
```

Comment:
these create mypy issues because we want the type to be not-None, but by default it's None